test/functional/web/safari-basic-e2e-specs.js

```diff
@@ -149,7 +150,7 @@ describe('Safari', function () {
       it('should be able to click links', async function () {
         let el = await driver.elementByLinkText('i am a link');
         await el.click();
-        await spinTitleEquals(driver, 'I am another page title');
+        await spinTitleEquals(driver, 'I am another page title - Sauce Labs');
```

Comment:
Might be worth just creating a `spinTitleStartWith` helper method, and do away with the different between Sauce and non-Sauce.

Reply:
This problem should be obsolete once we start using the local Appium server instead of the hosted one.